Library/Homebrew/utils/analytics.rb

```diff
@@ -11,7 +11,7 @@ def clear_os_prefix_ci
       def os_prefix_ci
         @os_prefix_ci ||= begin
           os = OS_VERSION
-          prefix = ", non-/usr/local" if HOMEBREW_PREFIX.to_s != "/usr/local"
+          prefix = ", custom-prefix" if HOMEBREW_PREFIX.to_s != HOMEBREW_DEFAULT_PREFIX
```

Comment:
This will mean all the previous analytics groupings are broken so probably needs to be tweaked to preserve the existing wording for macOS.